src/NUnitFramework/tests/Attributes/ApartmentAttributeTests.cs

```diff
@@ -47,6 +53,9 @@ public void TestWithRequiresSTARunsInSTA()
         [Test]
 #if THREAD_ABORT
         [Timeout(10000)]
+#endif
+#if PLATFORM_DETECTION && NETCOREAPP2_0
+        [Platform(Include = "Win")]
```

Comment:
All of these need to change to `"Win, Mono"` in case we ever run .NET Standard tests on Mono because Mono simulates apartment state.